cartographer/mapping/proto/trajectory_builder_options.proto

```diff
@@ -29,7 +29,6 @@ message InitialTrajectoryPose {
 message TrajectoryBuilderOptions {
   LocalTrajectoryBuilderOptions2D trajectory_builder_2d_options = 1;
   LocalTrajectoryBuilderOptions3D trajectory_builder_3d_options = 2;
-  bool pure_localization = 3;
```

Comment:
This breaks existing pbstreams. Is it worth it?

Reply:
We can make it backwards-compatible. I'll send the PR.